slack/rtm/client.py

```diff
@@ -194,7 +199,7 @@ def start(self) -> asyncio.Future:
             SlackApiError: Unable to retrieve RTM URL from Slack.
         """
         # TODO: Add Windows support for graceful shutdowns.
-        if os.name != "nt" and current_thread() == main_thread():
+        if os.name != "nt" and current_thread() == main_thread() and not self._disable_signals:
```

Comment:
I should have mentioned this earlier but what do you think about renaming to `enable_signal_handlers` and use the opposite value (default: True)? I would like all the flag options to be consistent - enabling something.

Reply:
@amckenna-pinterest I should have requested this earlier but I have a suggestion about the naming 🙇 Others look good to me.